docs/language-server/protocol-language-server.md

```diff
@@ -3918,6 +3971,363 @@ standard input.
 null;
 ```
 
+## Library-Related Operations
+
+The library-related operations provide the Language Server with capabilities to
+check and modify project's edition settings, list editions published in a given
+edition, create local library projects which can be imported in the currently
+opened project and publish them.
+
+### `editions/listAvailable`
+
+Lists editions available on the system.
+
+Moreover, if `update` is set to `true`, it will download any new editions from
+the repositories and include them in the result as well.
+
+> Currently, if `update` was `true` but some downloads failed, the endpoint will
+> still return a success, just containing the editions that were already
+> available. In the future it should emit warnings using proper notification
+> channels.
+
+#### Parameters
+
+```typescript
+{
+  update: Boolean;
+}
+```
```

Comment:
I'm thinking that it may be worth making this `update` optional too (default to `false`). This way it will be easier to modify the API in the future.